### ISSUE TITLE
TypeError: null is not an object (evaluating 'vnode.componentInstance') on Safari

### ISSUE THREAD

**Issue by reado** [ectoflow/vue-stripe-elements#95]
Hey Everyone

I just found on that I get a console error in Safari when I'm using a `Card` element in my component.

I have a pretty straight forward setup where I'm loading the Stripe V3 library in my server rendered page (in my case, that's a Laravel Blade Template) and all i'm trying to do is allow customers to update their credit card information using the `handleCardSetup` function.  This is working great on all other browsers but Safari hates this and the the debug console is hard to follow along.

I saw some other posts about Safari indicating that 'Stripe should be loaded in `mounted()`' which I'm assuming means removing the V3 script tag from my server rendered page, and instead injecting it into the mounting method.

I did this but i'm still getting a ton of errors in the console indicating that the V3 library hasn't loaded yet.

Anyone have any suggestions?

**Comment by softbeehive** [comment]
Should be fixed by #120, feel free to reopen if it isn't the case
